project_parent/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Project Parent',
+    'summary': """
+        Project Patent.""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    "author": "Elico Corp, Odoo Community Association (OCA)",
+    'website': 'https://www.elico-corp.com/',
+    'depends': [
+        'account_analytic_parent',
```

Comment:
Don't depend on this one and create your own field parent_project_id as many2one reference to project.